<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="frame" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#frame{
    
	background-color: qlineargradient(spread:pad, x1:0.467, y1:0.37, x2:1, y2:1, stop:0 rgba(0, 80, 138, 255), stop:1 rgba(255, 255, 255, 255));

}</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 20pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Make your unforgettable train experience with us!</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTrainButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>650</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 63 12pt &quot;Sitka Small Semibold&quot;;
color:rgb(0, 86, 148);
}

QPushButton:hover{
	background-color: #00508a;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Add Train</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputCapacity">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>780</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background-color:qlineargradient(spread:pad, x1:0.467, y1:0.37, x2:1, y2:1, stop:0 rgba(0, 80, 138, 255), stop:1 rgba(255, 255, 255, 255));
font: 63 12pt &quot;Sitka Small Semibold&quot;;
color:white;
}

QPushButton:hover{
	background-color: white;
	color: #00508a;
}</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="capacityLabel_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNumofcart">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="numofcartLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Number of cart</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputManufacturer">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="manufacturerLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Manufacturer</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>720</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Sitka Heading Semibold&quot;;
color:rgb(180, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="activeRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 10pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="inactiveRadioButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 10pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Inactive</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
